### PR TITLE
Ddcnl 3475

### PR DESCRIPTION
**Pull request by Dandyboyo2**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -68,9 +68,6 @@ error.divorce-reason.required=Select an answer for keeping your Marriage Allowan
 pages.form.field-required-alt.endReason=Select an answer for keeping your Marriage Allowance until the end of the tax year
 
 pages.form.extra-year.field-required=Tell us whether you would like to apply for the previous {0} to {1} tax year
-#TODO remove after 6 April 2020
-pages.form.field-required.year-2015-=Confirm whether you would like to apply for the previous 2015 to 2016 tax year
-pages.form.field-required.year-2016-=Confirm whether you would like to apply for the previous 2016 to 2017 tax year
```
Comment (kurtlogan) [inline conf/messages:73]: Don't think this line should be removed as this is cy-4

File: conf/messages.cy [hunk 2]
```diff
@@ -57,13 +57,11 @@ pages.form.field-required.partners-income=Dewiswch ‘Iawn’ os yw incwm eich p
...
-pages.form.field-required.multiyear-transferor-income-criteria=Rhowch wybod a oedd eich incwm blynyddol er 6 Ebrill 2016 ar lefel y Lwfans Personol neu’n is.
-pages.form.field-required.multiyear-recipient-income-criteria=Rhowch wybod a yw’ch partner yn talu treth ar y gyfradd sylfaenol er 6 Ebrill 2016?
+pages.form.field-required.multiyear-transferor-income-criteria=Rhowch wybod a oedd eich incwm blynyddol er 6 Ebrill 2017 ar lefel y Lwfans Personol neu’n is.
+pages.form.field-required.multiyear-recipient-income-criteria=Rhowch wybod a yw’ch partner yn talu treth ar y gyfradd sylfaenol er 6 Ebrill 2017?
 pages.form.field-required.applyForHistoricYears=Dewiswch Iawn a hoffech wneud cais ar gyfer blynyddoedd treth cynharach
 
 pages.form.extra-year.field-required=Rhowch wybod a hoffech wneud cais ar gyfer y flwyddyn dreth flaenorol, sef {0} i {1}
-pages.form.field-required.year-2015-=Cadarnhewch a hoffech wneud cais ar gyfer y flwyddyn dreth flaenorol, sef 2015 i 2016
-pages.form.field-required.year-2016-=Cadarnhewch a hoffech wneud cais ar gyfer y flwyddyn dreth flaenorol, sef 2016 i 2017
```
Comment (kurtlogan) [inline conf/messages.cy:66]: Same as above

**Review by kurtlogan — CHANGES_REQUESTED** [review]

File: conf/messages (hunk 1 above)
Comment (TheDanielDoyle) [reply conf/messages:73]: I've put the messages back in.

File: conf/messages.cy (hunk 2 above)
Comment (TheDanielDoyle) [reply conf/messages.cy:66]: I've put the messages back in.

File: conf/messages (hunk 1 above)
Comment (kurtlogan) [reply conf/messages:73]: Ah I only meant the second one, the first one should be removed still

Comment (TheDanielDoyle) [reply conf/messages:73]: Ah i see now. Removed the 2015 line.

**Review by kurtlogan — APPROVED** [review]

**Pull request merged**